src/hooks/useIsBlockToAddFeed.ts

```diff
@@ -0,0 +1,27 @@
+import useCardsList from '@hooks/useCardsList';
+import useOnyx from '@hooks/useOnyx';
+import {getCompanyFeeds, getSelectedFeed} from '@libs/CardUtils';
+import {isCollectPolicy} from '@libs/PolicyUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+import isLoadingOnyxValue from '@src/types/utils/isLoadingOnyxValue';
+import useCardFeeds from './useCardFeeds';
+import usePolicy from './usePolicy';
+
+function useIsBlockToAddFeed(policyID?: string) {
+    const policy = usePolicy(policyID);
+
+    const [cardFeeds, allFeedsResult] = useCardFeeds(policyID);
+    const [lastSelectedFeed] = useOnyx(`${ONYXKEYS.COLLECTION.LAST_SELECTED_FEED}${policyID}`, {canBeMissing: true});
+    const companyFeeds = getCompanyFeeds(cardFeeds);
```

Comment:
```suggestion
    const companyFeeds = getCompanyFeeds(cardFeeds, true);
```

Do we need to exclude pending delete feeds?